### ISSUE TITLE
Need help, Cannot find port

### ISSUE THREAD

**Issue by lonezeig420** [patrix87/PowerShellGSM#76]
context: ran a server fine for a while. Quit then came back and now the console is showing this error? I am confused because 27015 is not a port I use or have set. any thoughts?
CreateBoundSocket: ::bind couldn't find an open port between 27015 and 27015

**Comment by patrix87** [comment]
The server might be crashed on hanging in the background.
Reboot the VM or computer it's running on.

**Comment by lonezeig420** [comment]
I have tried that several times with no success unfortunately

**Comment by patrix87** [comment]
27015 usually is a steam query port.
If you have more than one server running you may need to change it to 27016 in the config file.

**Comment by lonezeig420** [comment]
Yes I do have an enshrouded server running also that hasn't had an issue. So how do I got about changing that like what config file if you don't mind?

**Comment by patrix87** [comment]
Enshrouded and Palworld might be using the same 27015 port.
I don't see any configuration for either of those games to change that port.
You may have to run both servers on a separated VM.

**Comment by lonezeig420** [comment]
That's weird cause I've run both at the same time for months with no issues.

**Comment by patrix87** [comment]
Palworld may have updated the game and started using the port for something. Usualy that port is a range and would auto-negociate and pick the next available port. Make sure you also pull the most recent code from powershellGSM and update your server config file from the template file. The launch file changed recently
